azure-devops/javascript/stages/50-deployment/k8s.yaml

```diff
@@ -17,13 +17,10 @@ stages:
               dockerRegistryEndpoint: '$(DOCKER_REGISTRY_ENDPOINT)'
               kubernetesServiceConnection: '$(K8S_ENDPOINT)'
               namespace: '$(K8S_NAMESPACE)'
-          - script: |
-              for var in $(compgen -e); do
-                escaped_value=$(printf '%s\n' "${!var}" | sed 's:[\/&]:\\&:g;$!s/$/\\/')
-                sed -i "s/\$($var)/$escaped_value/g" .ci/config/opensearch_dashboards.yml
-                sed -i "s/\${$var}/$escaped_value/g" .ci/config/opensearch_dashboards.yml
-              done
-            displayName: 'Replace Environment Variables'
+          - template: '../../../global/abstracts/replace-env-vars.yml'
+            parameters:
+              FILE: '.ci/config/opensearch_dashboards.yml'
+              displayName: 'Replace Environment Variables'
```

Comment:
So, you can see by this comment here https://github.com/rios0rios0/pipelines/pull/71/files#r1769330332, that this is not consistent.